eslint-plugin broken from 0.18.1

In an nx monorepo with `eslint@9.32.0`, and the following `eslint.config.mjs` file:

```js
import nx from '@nx/eslint-plugin';
import sheriff from '@softarc/eslint-plugin-sheriff';

export default [
  sheriff.configs.all,
  // ...everything else as nx provides it out of the box
];
```

And with the following `sheriff.config.ts` file:

```typescript
import { SheriffConfig } from '@softarc/sheriff-core';

export const config: SheriffConfig = {
  enableBarrelLess: true,
  modules: {
    'apps/whatever/src': 'app',
    'libs/foo/src': 'foo',
    'libs/bar/src': 'bar',
  },
  depRules: {
    'root': 'noTag',
    'noTag': 'noTag',
    'app': ['foo', 'bar'],
    'foo': ['foo'],
    'bar': ['bar'],
  },
};
```

When my lib under `libs/bar/src` contains an import from `foo` (which is not allowed according to the depRules above), then I would expect a lint error.

This worked as expected with the version `0.18.0` but after updating to `0.19.0` it no longer errors.

Working backwards, it looks like it first broke in version `0.18.1`.

Also, any `// eslint-disable-next-line @softarc/sheriff/dependency-rule` comments now cause the following error when linting:

```
error  Definition for rule '@softarc/sheriff/dependency-rule' was not found  @softarc/sheriff/dependency-rule
```

I see the same issue here. No rules seem to be applied. I have simplified our configuration to only one `eslint.config.mjs` file:

```mjs
import { configs } from '@softarc/eslint-plugin-sheriff';
export default [configs.all];
```

Running ` npx eslint --debug apps/eusoda/src/main.ts ` shows no rules applied:
```
 npx eslint --debug apps/eusoda/src/main.ts 
  eslint:cli CLI args: [ '--debug', 'apps/eusoda/src/main.ts' ] +0ms
  eslint:cli Using flat config? true +1ms
  eslint:cli Running on files +1ms
  eslint:eslint Using config loader LegacyConfigLoader +0ms
  eslint:eslint Using file patterns: apps/eusoda/src/main.ts +0ms
  eslint:eslint Deleting cache file at ~/monorepo/.eslintcache +0ms
  eslint:eslint 1 files found in: 0ms +0ms
  eslint:config-loader Calculating config for file ~/monorepo/apps/eusoda/src/main.ts +0ms
  eslint:config-loader Searching for eslint.config.js +0ms
  eslint:config-loader [Legacy]: Calculating config for ~/monorepo/apps/eusoda/src/main.ts +1ms
  eslint:config-loader [Legacy]: Using config file ~/monorepo/eslint.config.mjs and base path ~/monorepo +0ms
  eslint:config-loader Calculating config array from config file ~/monorepo/eslint.config.mjs and base path ~/monorepo +0ms
  eslint:config-loader Loading config file ~/monorepo/eslint.config.mjs +0ms
  eslint:config-loader Loading config from ~/monorepo/eslint.config.mjs +0ms
  eslint:config-loader Config file URL is file://~/monorepo/eslint.config.mjs +0ms
  eslint:config-loader Config file ~/monorepo/eslint.config.mjs is not empty +45ms

~/monorepo/apps/eusoda/src/main.ts
  0:0  warning  File ignored because no matching configuration was supplied
```

--- a/packages/eslint-plugin/src/lib/configs/all.ts
+++ b/packages/eslint-plugin/src/lib/configs/all.ts
@@ -2,7 +2,7 @@ import type { TSESLint } from '@typescript-eslint/utils';
 import rules from '../rules';
 
 const commonConfig: TSESLint.FlatConfig.Config = {
-  files: ['*.ts, *.js'],
+  files: ['**/*.ts', '**/*.js'],
   ignores: ['sheriff.config.ts'],
   languageOptions: {
     sourceType: 'module',

